pkg/controllers/edgeconnect/controller.go

```diff
@@ -26,20 +30,23 @@ import (
 )
 
 const (
-	errorUpdateInterval   = 1 * time.Minute
-	defaultUpdateInterval = 30 * time.Minute
+	fastUpdateInterval    = 1 * time.Minute
+	defaultUpdateInterval = 3 * time.Minute // 30
```

Comment:
what is with the `//30`? (probably a mistake)

Reply:
3 is a debug value, fixed